runtime/src/iree-amd-aie/driver/xrt/native_executable.cc

```diff
@@ -184,8 +184,19 @@ iree_status_t iree_hal_xrt_native_executable_create(
       return iree_make_status(IREE_STATUS_INTERNAL, "XCLBIN load error: %s",
                               e.what());
     }
-    device->register_xclbin(*xclbin);
-    xrt::hw_context context(*device, xclbin->get_uuid());
+    try {
+      device->register_xclbin(*xclbin);
+    } catch (std::runtime_error& e) {
```

Comment:
```suggestion
    } catch (const std::exception & e) {
```
Super pedantic would be to then `catch (...)` after that https://en.cppreference.com/w/cpp/language/catch (not a request!)

Reply:
yea no chance i'm doing that lol